### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by wagnerand**
Test passing depends on #1250.

### CODE REVIEW

File: docs/repositories.rst [hunk 1]
```diff
@@ -6,13 +6,13 @@ mozilla-central
 
 Browser code is maintained in mozilla-central. This includes several areas related to add-ons. All of the following bug trackers are in Bugzilla:
 
-Add-ons manager
+Add-ons Manager
 ~~~~~~~~~~~~~~~
-Handles installing, running and updating add-ons within the application (eg: Firefox). Also has pages like about:addons. In bugzilla - product: Toolkit, component: Add-ons Manager. `Existing bugs <https://bugzilla.mozilla.org/buglist.cgi?bug_status=UNCONFIRMED&bug_status=NEW&bug_status=ASSIGNED&bug_status=REOPENED&component=Add-ons%20Manager&product=Toolkit>`__ or `file a new bug <https://bugzilla.mozilla.org/enter_bug.cgi?product=Toolkit&component=Add-ons%20Manager>`__.
+Handles installing, running and updating add-ons within the application (eg: Firefox). Also has pages like about:addons. In bugzilla - product: Toolkit, component: Add-ons Manager. `Existing bugs <https://bugzilla.mozilla.org/buglist.cgi?bug_status=UNCONFIRMED&bug_status=NEW&bug_status=ASSIGNED&bug_status=REOPENED&component=Add-ons%20Manager&product=Toolkit&list_id=15432463>`__ or `file a new bug <https://bugzilla.mozilla.org/enter_bug.cgi?product=Toolkit&component=Add-ons%20Manager>`__.
```
Comment (muffinresearch) [inline docs/repositories.rst:11]: Are list ids required? I see a different one if I start the request without it, which makes me suspect they are generated

File: docs/l10n.rst [hunk 2]
```diff
@@ -163,11 +163,11 @@ E.g: ``https://addons-dev.allizom.org/en-US/firefox/`` becomes
...
-.. _`dev`: https://addons-dev.allizom.org
-.. _`stage`: https://addons.allizom.org
+.. _`dev`: https://addons-dev.allizom.org/
+.. _`stage`: https://addons.allizom.org/
 .. _`potools`: https://github.com/mozilla/potools/
 .. _`Pontoon`: https://pontoon.mozilla.org/
 .. _`addons-server on Pontoon`: https://pontoon.mozilla.org/projects/amo/info/
 .. _`addons-frontend on Pontoon`: https://pontoon.mozilla.org/projects/amo-frontend/info/
-.. _`new profile`: https://support.mozilla.org/kb/profile-manager-create-and-remove-firefox-profiles
+.. _`new profile`: https://support.mozilla.org/en-US/kb/profile-manager-create-remove-switch-firefox-profiles
```
Comment (muffinresearch) [inline docs/l10n.rst:172]: Should this have the locale removed to allow for redirs based on accept-language?

File: docs/repositories.rst [hunk 3]
```diff
@@ -26,20 +26,26 @@ Bug tracker is in GitHub and can be used for almost anything add-ons related. `E
...
+addons-code-manager
+~~~~~~~~~~~~~~~~~~~
+A web application to manage add-on source code, such as reviewing code for add-ons submitted to addons.mozilla.org. The `repository <https://github.com/mozilla/addons-code-manager>`__ and `issue tracker <https://github.com/mozilla/addons-code-manager/issues/>`__ is on GitHub.
 
 addons-linter
 ~~~~~~~~~~~~~
-The new JavaScript version of the validator for the `addons.mozilla.org <https://addons.mozilla.org>`__ website. The `repository <https://github.com/mozilla/addons-linter>`__ is on GitHub. The bug tracker is `GitHub <https://github.com/mozilla/addons-linter/issues/>`__. Documentation is `on readthedocs <https://addons-server.readthedocs.io>`__. See also `dispensary <https://github.com/mozilla/dispensary>`__.
+The linter is linting WebExtensions for `addons.mozilla.org <https://addons.mozilla.org/>`__ and `web-ext <https://github.com/mozilla/web-ext/>`__. The `repository <https://github.com/mozilla/addons-linter>`__, `issue tracker <https://github.com/mozilla/addons-linter/issues/>`__ and `documentation <https://mozilla.github.io/addons-linter/>`__ is on GitHub.
```
Comment (muffinresearch) [inline docs/repositories.rst:39]: I wonder if `linting` should be expanded into a description of what that means in this case?

**Review by muffinresearch — APPROVED** [review]
Thanks for the clean-up.

r+wc, once doc tests pass.

File: docs/repositories.rst (hunk 1 above)
Comment (wagnerand) [reply docs/repositories.rst:11]: Ah, no. I was trying to make bugzilla links work in the linkchecker, so I played around with various variations, without luck, until I realized that the buglist page won't load without a cookie.

File: docs/l10n.rst (hunk 2 above)
Comment (wagnerand) [reply docs/l10n.rst:172]: Good catch!

File: docs/repositories.rst (hunk 3 above)
Comment (wagnerand) [reply docs/repositories.rst:39]: Do you have a specific copy in mind?

File: docs/repositories.rst (hunk 1 above)
Comment (wagnerand) [reply docs/repositories.rst]: 4c95ce9

File: docs/l10n.rst (hunk 2 above)
Comment (wagnerand) [reply docs/l10n.rst]: 4c95ce9

**Comment by wagnerand** [comment]
Rebased to make tests pass.

File: docs/repositories.rst (hunk 3 above)
Comment (muffinresearch) [reply docs/repositories.rst:39]: I was thinking something like "The linter checks web extensions for common errors and potential problems".

Comment (wagnerand) [reply docs/repositories.rst]: Thank you, added in f2a597a.

**Pull request merged**